docs/install/remnawave-node.md

```diff
@@ -67,12 +67,22 @@ docker compose up -d && docker compose logs -f -t
 
 ## Advanced usage
 
-### GeoSite files
+### Loading modified geosite and geoip files
 
-You can mount additional geosite files into the `/usr/local/share/xray/` directory in the container.
+:::info
+WARNING!
+1. This manual does not contain mandatory instructions, it is only an example of implementation without reference to the file `*-zapret.dat` used in it. In your implementation, it will not be appropriate to blindly repeat the call ext:*-zapret.dat:zapret, since in reality the categories `:zapret` will be different. If you blindly, without understanding what is happening, repeat what is described, you will break your Xray configuration and get an error connecting to the node.
```

Comment:
> In your implementation, it will not be appropriate to blindly repeat the call ext:*-zapret.dat:zapret, since in reality the categories `:zapret` will be different. If you blindly, without understanding what is happening, repeat what is described, you will break your Xray configuration and get an error connecting to the node.
Думаю стоит укоротить и переформулировать. Слишком дословный перевод на англ.